packages/mime-types/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 <!-- ## Unreleased -->
 
+## 1.0.4 - 2021-09-02
+
+### Changed
```

Comment:
This is technically an "Added" because we're adding a new feature (or extending an existing feature). We'll add the minor version bump + date heading when we're publishing. So this should do:

```suggestion
### Added
```